app/views/oauth_clients/index.html.erb

```diff
@@ -3,34 +3,42 @@
 <% end %>
 
 <% unless @tokens.empty? %>
-<h3><%= t'oauth_clients.index.my_tokens' %></h3>
-<p><%= t'oauth_clients.index.list_tokens' %></p>
-<table>
-  <tr><th><%= t'oauth_clients.index.application' %></th>
-    <th><%= t'oauth_clients.index.issued_at' %></th><th>&nbsp;</th></tr>
-  <% @tokens.each do |token|%>
-    <%= content_tag_for :tr, token do %>
-      <td><%= link_to token.client_application.name, token.client_application.url %></td>
-      <td><%= token.authorized_at %></td>
+<div id="authorised_applications">
+  <h3><%= t'oauth_clients.index.my_tokens' %></h3>
+  <p><%= t'oauth_clients.index.list_tokens' %></p>
+  <table>
+    <tr>
+      <th><%= t'oauth_clients.index.application' %></th>
+      <th><%= t'oauth_clients.index.first_authorised' %></th>
+      <th>&nbsp;</th>
+    </tr>
+    <% @tokens.group_by { |t| t.client_application }.each do |client,tokens|%>
```

Comment:
Will the order be random or sorted by name? Is it possible to add an `order` statement to sort by "First authorized"?